source/prerequisites.rst

```diff
@@ -45,6 +45,8 @@ Here is a virtual host configuration example for ``Apache 2`` web server.
             # Redirect all requests to GLPI router, unless file exists.
             RewriteCond %{REQUEST_FILENAME} !-f
             RewriteRule ^(.*)$ index.php [QSA,L]
+            # If you are going to use 'Alias' directive shown above, you need set RewriteBase:
+            # RewriteBase /glpi
```

Comment:
I did not had to use it when I made tests on this part. Are your rewrite directives placed inside the vhost configration file, or inside a `.htaccess` file ?

Reply:
My rewrite directives are placed inside vhost configuration. I've tried several ways, this is the only one I've had success with.